packages/lou_bank_ui/lou_bank_gallery/test/widget_test.dart

```diff
@@ -0,0 +1,11 @@
+// This is a basic Flutter widget test.
+//
+// To perform an interaction with a widget in your test, use the WidgetTester
+// utility that Flutter provides. For example, you can send tap and scroll
+// gestures. You can also use WidgetTester to find child widgets in the widget
+// tree, read text, and verify that the values of widget properties are correct.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {}
```

Comment:
missing testes :)

Reply:
I will add them after creating the theme